kinds/teamrole/teamrole_kind.cue

```diff
@@ -0,0 +1,28 @@
+package kind
+
+import "time"
+
+name:        "TeamRole"
+maturity:    "merged"
+description: "An association between a Team and a Role"
+
+lineage: seqs: [
+	{
+		schemas: [
+			// v0.0
+			{
+				// Namespace aka tenant/org id.
+				namespace: string
+
+				// Unique role uid.
+				roleUid: string
+
+				// Unique team name
+				teamName: string
+
+				// Created indicates when the team role was created.
+				created: string & time.Time
```

Comment:
the inconsistency in these fields really bothers me, it seems like we should standardize handling for all these schematized kinds and have consistent meta fields like createdAt, createdBy, updatedAt, updatedBy, and uid.

Reply:
I vote we ignore all the common metadata until we have a way to set that in the k8s metadata